static/js/components/GroupShareSelect.jsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import Select from "@material-ui/core/Select";
+import MenuItem from "@material-ui/core/MenuItem";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormControl from "@material-ui/core/FormControl";
+import Chip from "@material-ui/core/Chip";
+import Input from "@material-ui/core/Input";
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    marginTop: theme.spacing(1),
+    minWidth: 120,
+  },
+  chips: {
+    display: "flex",
+    flexWrap: "wrap",
+  },
+  chip: {
+    margin: 2,
+  },
+}));
+
+const getStyles = (groupID, groupIDs = [], theme) => ({
+  fontWeight:
+    groupIDs.indexOf(groupID) === -1
+      ? theme.typography.fontWeightRegular
+      : theme.typography.fontWeightMedium,
+});
+
+const GroupShareSelect = ({
```

Comment:
At least a few other components could import this now, too, right?

Reply:
Indeed

Reply:
@dannygoldstein Can you make an issue so we can track this?

Reply:
done